pybase/region/client.py

```diff
@@ -294,7 +293,7 @@ def NewClient(host, port, pool_size, secondary=False):
         c.pool_size = pool_size
         for x in range(pool_size):
             s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
-            s.connect((host, int(port)))
+            s.connect((c.host, int(port)))
```

Comment:
use `c.host` instead of the passed in param `host` since we now handle converting host into unicode in the client init

Reply:
yeah it doesn't matter since port is just 0-9 values, but I can change that if that's more thorough